doc/UsersGuide/BuildingRunningTesting/BuildSRW.rst

```diff
@@ -394,9 +419,7 @@ Next, users must source the Lmod setup file, just as they would on other systems
    module load build_<platform>_gnu
    export LDFLAGS+=" -L${MPI_ROOT}/lib "
```

Comment:
Suggesting to remove as it is no longer true:
```
export LDFLAGS+=" -L${MPI_ROOT}/lib "
The last line is primarily needed for the MacOS platforms.
```

Reply:
Agree, but this whole section (not just this line) will be removed in another PR, so I will not handle it here. PR #1234 removes it from the release branch, and there is a procedure to bring any relevant changes that went directly into the release over to develop once the release is finalized.